file-formats/smbc/type/zif_aff_smbc_v1.intf.abap

```diff
@@ -118,68 +118,83 @@ INTERFACE zif_aff_smbc_v1
+      "! Enable Select all checkbox
+      select_all                    TYPE abap_bool,
       "! <p class="shorttext synchronized">Row Count Mode</p>
       "! Defines how the table handles the visible rows in the table
       "! $values {@link zif_aff_smbc_v1.data:co_row_count_mode}
       "! $default {@link zif_aff_smbc_v1.data:co_row_count_mode.fixed}
-      row_count_mode         TYPE string,
+      row_count_mode                TYPE string,
       "! <p class="shorttext synchronized">Row Count</p>
       "! Number of visible rows of the table
       "! $minimum 1
       "! $maximum 1000
       "! $default '5'
-      row_count              TYPE i,
+      row_count                     TYPE i,
+      "! <p class="shorttext synchronized">Frozen Column Count</p>
+      "! You can freeze the first columns of a table so that they always remain visible when scrolling horizontally
+      "! $minimum 0
+      "! $maximum 30
+      frozen_column_count           TYPE i,
       "! <p class="shorttext synchronized">Table Creation Mode</p>
       "! $values {@link zif_aff_smbc_v1.data:co_creation_mode_name}
       "! $default {@link zif_aff_smbc_v1.data:co_creation_mode_name.new_page}
-      creation_mode_name     TYPE ty_creation_mode-name,
+      creation_mode_name            TYPE ty_creation_mode-name,
       "! <p class="shorttext synchronized">Create at End</p>
       "! If true, a new row is added to the end of the table instead of the beginning of the table
-      create_at_end          TYPE ty_creation_mode-create_at_end,
+      create_at_end                 TYPE ty_creation_mode-create_at_end,
       "! <p class="shorttext synchronized">Hide Filter</p>
       "! If true, the user cannot filter data of the table
-      hide_filter            TYPE ty_personalization-hide_filter,
+      hide_filter                   TYPE ty_personalization-hide_filter,
       "! <p class="shorttext synchronized">Hide Sort</p>
       "! If true, the user cannot sort the table
-      hide_sort              TYPE ty_personalization-hide_sort,
+      hide_sort                     TYPE ty_personalization-hide_sort,
       "! <p class="shorttext synchronized">Hide Column</p>
       "! If true, the user cannot add and remove columns to the table
-      hide_column            TYPE ty_personalization-hide_column,
+      hide_column                   TYPE ty_personalization-hide_column,
       "! <p class="shorttext synchronized">Condensed Table Layout</p>
       "! If true, display rows in a condensed way. Only applicaple to Table Type 'GridTable'.
-      condensed_table_layout TYPE abap_bool,
+      condensed_table_layout        TYPE abap_bool,
+      "! <p class="shorttext synchronized">Include column headers in width calculation</p>
+      "! If true, include the column labels while calculating the default column width.
+      "! By default, the column width is calculated based on the type of the content
+      width_including_column_header TYPE abap_bool,
```

Comment:
Minor: Title does not match field name. I don't have a better proposal, but maybe others

Reply:
Field name & description are coming from here: https://sapui5.hana.ondemand.com/sdk/#/api/sap.fe.macros.Table%23controlProperties

Field names of the SMBC object are always equal to their FE counterpart attributes to simplify mapping

![image](https://github.com/user-attachments/assets/0dcbb018-0a66-4ee1-ade0-6b021d1b328d)